packages/devtools/client/pages/modules/storage.vue

```diff
@@ -0,0 +1,136 @@
+<script setup lang="ts">
+import JsonEditorVue from 'json-editor-vue'
+import 'vanilla-jsoneditor/themes/jse-theme-dark.css'
+
+definePageMeta({
+  icon: 'carbon-data-base',
+  title: 'Storage',
+})
+
+const router = useRouter()
+const searchString = ref('')
+const newKey = ref('')
+const current = ref()
+const fileKey = computed(() => useRoute().query?.key as string | undefined)
+
+const { data: keys, refresh } = await useAsyncData('storage', () => rpc.getStorageKeys())
```

Comment:
Instead of this, we could also use unstorage itself with a compatible HTTP driver. I will explain later today in call. (we can move forward and change later)

Reply:
Let's wait for our call to finalize this :)